internal/lattice/lattice.go

```diff
@@ -171,7 +171,7 @@ func (la *Lattice) Build(inp string) {
 			for i, w := pos, 0; i < endPos; i += w {
 				_, w = utf8.DecodeRuneInString(inp[i:])
 				end := i + w
-				dup, _ := la.dic.UnkIndexDup[int32(class)]
+				dup := la.dic.UnkIndexDup[int32(class)]
```

Comment:
__[Go Meta Linter]__

[unconvert] invalid operation: la.dic (variable of type *invalid type) has no field or method UnkIndexDup


<!-- Comment by [SideCI](https://sideci.com) -->